include/exiv2/error.hpp

```diff
@@ -202,16 +270,20 @@ namespace Exiv2 {
         //! @name Creators
         //@{
         //! Constructor taking only an error code
-        EXV_DLLLOCAL explicit BasicError(int code);
+        EXV_DLLLOCAL explicit BasicError(ErrorCodes code);
```

Comment:
[suggestion] Normally the enum types do not have a plural name (`ErrorCodes`) but singular (`ErrorCode`). What we are passing to the function is a single error code, not a list of them. What do you think about changing the enum name ?

Reply:
Most of the enums in the source seem to use the singular for their name, so I'll change this for consistency. Thanks for bringing that up, it makes a lot more sense in the function signatures.